ARTstor.js

```diff
@@ -1,22 +1,25 @@
 {
-	"translatorID": "5278b20c-7c2c-4599-a785-12198ea648bf",
-	"label": "ARTstor",
-	"creator": "Sebastian Karcher",
-	"target": "^https?://library\\.artstor\\.org",
-	"minVersion": "3.1",
-	"maxVersion": "",
```

Comment:
Like I said, `"https?://([^/]+\\.)?(artstor|sscommons)\\.org\/(open)?library"` is what you want. If that doesn't work, please post a URL that you're trying to test against.